Python/Include/Arithmetics.hpp

```diff
@@ -14,6 +14,38 @@ namespace Rover::Details {
   inline constexpr bool contain_objects_v = contain_objects<Args...>::value;
 
   const pybind11::module& import_operator();
+
+  pybind11::object apply_operator(const char* operator_name, const
+    pybind11::object& lhs, const pybind11::object& rhs);
+
+  template<typename T1, typename T2>
+  struct OperatorApplicator;
+
+  template<typename T>
+  struct OperatorApplicator<T, pybind11::object> {
+    pybind11::object operator()(const char* operator_name, const T& lhs,
```

Comment:
With the introduction of ```std::string_view```, I generally prefer that it always be used, even if you know you'll only be using ```const char*```.

Reply:
`std::string_view` does not allow for any direct conversion to a C-string due to the various types of strings it can represent.